portal/src/main/webapp/content/examples.html

```diff
@@ -1,15 +1,23 @@
-<p>
```

Comment:
Changes in this file are just to convert these link tags into an unordered list for easier styling.